framework/PostProcessors/TopologicalDecomposition.py

```diff
@@ -32,6 +33,18 @@
 import Runners
 #Internal Modules End-----------------------------------------------------------
 
+def _toStr(s):
+  """
+    Removes unicode from strings in Python 2 so amsc can use it.
+    @ In, s, unicode or str, String to convert to plain str
+    @ Out, s, str, Converted str
+  """
+  if sys.version_info.major > 2:
+    return s
+  return s.encode('ascii')
```

Comment:
shouldn't we put this in the utils.py module so other modules can use it in case needed?

Reply:
Actually, I can remove it in all three cases by removing from __future__ import unicode_literals.